### PR TITLE
BQL examples

### PR DESCRIPTION
**Pull request by 0aix**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/ws/examples.md [hunk 1]
```diff
@@ -1004,6 +1042,97 @@ subtract 24 from it, you get the lower bound of the true count.
 Note that this also means the order of the items could be off. If two items had ```Count``` within 24 of each other, it is possible that the higher one *may* actually have had a true count *lower* than
 the second one and possibly be ranked higher. There is no such situation in this result set.
 
+### Lateral View Explode
```
Comment (akshaisarma) [inline docs/ws/examples.md:1045]: Add an example for multiple lateral view explodes

File: docs/ws/examples.md [hunk 2]
```diff
@@ -1004,6 +1042,97 @@ subtract 24 from it, you get the lower bound of the true count.
...
+         "score": 100
+      }
+   ],
+   "meta": "<EDITED OUT>"
+}
+```
+
+Lateral view explode can be used to also explode lists to a single field. Multiple lateral view explodes can also be chained in the same query.
+
+### Outer Query
```
Comment (akshaisarma) [inline docs/ws/examples.md:1108]: Mention what can't be done here

Comment (0aix) [reply docs/ws/examples.md:1120]: I'm not sure what you mean

Comment (akshaisarma) [reply docs/ws/examples.md:1120]: Windowing only I guess

Comment (0aix) [reply docs/ws/examples.md:1120]: It's there under the query. 👀

Comment (akshaisarma) [reply docs/ws/examples.md:1120]: Oh must have missed it

**Review by akshaisarma — APPROVED** [review]

**Pull request merged**